src/main/resources/io/viash/runners/nextflow/states/findStates.nf

```diff
@@ -82,8 +82,8 @@ def findStates(Map params, Map config) {
           // construct renameMap
           if (args.rename_keys) {
             def renameMap = args.rename_keys.collectEntries{renameString ->
-              def split = renameString.split(";")
-              assert split.size() == 2: "Argument 'rename_keys' should be of the form 'newKey:oldKey,newKey:oldKey'"
+              def split = renameString.split(":")
+              assert split.size() == 2: "Argument 'rename_keys' should be of the form 'newKey:oldKey;newKey:oldKey'"
```

Comment:
Does it make sense that the assert suggest using a single string with the multiple separator? I find it a bit confusing.
I believe the main thing the assert message wants to convey is that the notation is `'newKey:oldKey` and not e.g. `'newKey,oldKey'`. 
If there are multiple keys to be renamed, that is a secondary concern.

so maybe change it to something like:
```suggestion
assert split.size() == 2: "Argument 'rename_keys' should be of the form 'newKey:oldKey', or 'newKey:oldKey;newKey:oldKey' in case of multiple values"
```

The examples above use the array notation `["newKey1:oldKey1", "newKey2:oldKey2"]`.